packages/request-pre/src/util/mock.ts

```
@@ -0,0 +1,13 @@
+import { ApiSchema } from '../api';
+import { requestReturn } from '../util/request';
+export default function addMock(requestInfo: ApiSchema): requestReturn | void {
+    const { config, mock } = requestInfo;
+    const mockStatus = config && config.mock;
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)